packaging/installer/methods/kubernetes.md

```diff
@@ -0,0 +1,138 @@
+<!--
+title: "Install Netdata on a Kubernetes cluster"
+description: "Use Netdata's Helm chart to bootstrap a Netdata monitoring and troubleshooting toolkit on your Kubernetes (k8s) cluster."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/packaging/installer/methods/kubernetes.md
+-->
+
+# Install Netdata on a Kubernetes cluster
+
+This document details how to install Netdata on an existing Kubernetes (k8s) cluster. By following these directions, you
+will use Netdata's [Helm chart](https://github.com/netdata/helmchart) to bootstrap a Netdata deployment on your cluster.
+The Helm chart installs one parent pod for storing metrics and managing alarm notifications plus an additional child pod
+for every node in the cluster.
+
+Each child pod will collect metrics from the node it runs on in addition to [22 supported
+services](https://github.com/netdata/helmchart#service-discovery-and-supported-services) via [service
+discovery](https://github.com/netdata/agent-service-discovery/). Each child pod will also collect
+[cgroups](/collectors/cgroups.plugin/README.md),
+[Kubelet](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubelet), and
+[kube-proxy](https://learn.netdata.cloud/docs/agent/collectors/go.d.plugin/modules/k8s_kubeproxy) metrics from its node.
+
+To install Netdata on a Kubernetes cluster, you need:
+
+-   A working cluster running Kubernetes v1.9 or newer.
+-   The [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/) command line tool, within [one minor version
+    difference](https://kubernetes.io/docs/tasks/tools/install-kubectl/#before-you-begin) of your cluster, on an
+    administrative system.
+-   The [Helm package manager](https://helm.sh/) v3.0.0 or newer on the same administrative system.
+
+The default configuration creates four pods in your cluster: three `child` pods and one `parent` pod. The `child` pods
```

Comment:
child is [daemonset](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/)

> A DaemonSet ensures that all (or some) Nodes run a copy of a Pod. As nodes are added to the cluster, Pods are added to them. As nodes are removed from the cluster, those Pods are garbage collected. Deleting a DaemonSet will clean up the Pods it created.

> three `child` pods

So it is _number of nodes_ children. 

Every node has netdata child - one netdata per node.

And only one node has 2 netdatas - parent + child.

Reply:
Thanks for noticing this. I forgot to change this part from when I was talking about a specific 3-node cluster. Changed it up to be clearer that there's a child for every node, no matter how many nodes there are.